user_guide.adoc

```diff
@@ -1377,30 +1377,82 @@ library" for expression language.
 |Return type
 |Description
 
-|to_s
+|`to_s`
 |String
 |Converts integer into a string using decimal representation
-
-|to_s(radix)
```

Comment:
That's right, but it only seems to reach that to_s (with radix argument) point in the code if the type which the method is called on is BytesType.

To work with integers, wouldn't we need to add the following:
  case (_: IntType, "to_s") => bytesToStr(translate(obj), args(0))

And then update the various translators to handle conversion of integers into strings with various radixes. For example, CppTranslator.scala currently only implements integer->string conversion if the radix is 10.